app/src/componentLibrary/fieldText/fieldText.jsx

```diff
@@ -55,7 +55,12 @@ export const FieldText = ({
 
   return (
     <>
-      {label && <span className={cx(VARIANT, 'label', { disabled })}>{label}</span>}
+      {label && (
+        <span className={cx(VARIANT, 'label', { disabled })}>
+          {label}
+          {isRequired && <span className={cx('asterisk')}>*</span>}
```

Comment:
Do we need to support case when we have label with asterisk and placeholder with asterisk?
Now it is works as described above.

Reply:
We need to support placeholder for attributes asterisk